### ISSUE TITLE
Whitespace at end of commented line is not reported

### ISSUE THREAD

**Issue by harleypig** [jhinch/nginx-linter#14]
A commented line ending with whitespace is not reported when 'trailing-whitespace' is true.

```
# some explanatory text with whitespace at the end____
worker_processes auto;___
```
In the example above, underlines ('_') are spaces. The commented line will not be reported, while the uncommented will be.

I'm assuming you ignore commented lines right off the bat, so I'm not sure if anything can be done about this.

**Comment by jhinch** [comment]
Comments are part of the [AST](https://github.com/jhinch/nginx-linter/blob/master/lib/parser/parser.js#L133) so this should be easy fix. It should be a matter of adding a case to the switch statement in [rule-trailing-whitespace.js](https://github.com/jhinch/nginx-linter/blob/master/lib/rules/rule-trailing-whitespace.js#L6) to get it to work.

**Comment by jhinch** [comment]
Fixed in 0.6.0
